packages/test-env-helpers/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@inrupt/test-env-helpers",
+  "version": "1.0.0",
```

Comment:
For the versioning, everything in this repo versions together, so the latest would be `1.3.1` iirc.